workflow-dockstore-metrics/README.md

```diff
@@ -0,0 +1,14 @@
+The `pull_and_send_metrics.py` script pulls workflow execution data from BigQuery and sends it to Dockstore.
+It then updates the metadata table to indicate that the data has been sent to Dockstore.
+
+Requires:
```

Comment:
Could you add a section about testing, and especially the steps to gain access to their QA environment, and the corresponding urls to change and where to change them? (I am assuming the config.ini file)